src/Microsoft.ML.OnnxTransformer/Microsoft.ML.OnnxTransformer.csproj

```diff
@@ -9,8 +9,8 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
-    <ProjectReference Include="..\Microsoft.ML.OnnxConverter\Microsoft.ML.OnnxConverter.csproj" />
     <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="$(MicrosoftMLOnnxRuntimePackageVersion)" />
+    <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPackageVersion)" />
```

Comment:
You’ll need to add this reference to the pkg nupkgproj.

Reply:
Ok.